src/views/VideoView/VideoView.tsx

```diff
@@ -128,81 +129,83 @@ const VideoView: React.FC = () => {
   }
 
   return (
-    <Container>
-      <PlayerWrapper>
-        <PlayerContainer>
-          {video ? (
-            <VideoPlayer
-              playing={playing}
-              src={video.media.location}
-              fill
-              posterUrl={video.thumbnailUrl}
-              onEnd={handleVideoEnd}
-              onTimeUpdated={handleTimeUpdate}
-              onPlay={handlePlay}
-              onPause={handlePause}
-              startTime={startTimestamp}
-            />
-          ) : (
-            <PlayerPlaceholder />
-          )}
-        </PlayerContainer>
-      </PlayerWrapper>
-      <InfoContainer>
-        {video ? <TitleText variant="h2">{video.title}</TitleText> : <Placeholder height={46} width={400} />}
-        <Meta variant="subtitle1">
-          {video ? (
-            formatVideoViewsAndDate(video.views || null, video.createdAt, { fullViews: true })
-          ) : (
-            <Placeholder height={18} width={200} />
-          )}
-        </Meta>
-        <ChannelContainer>
-          <ChannelLink id={video?.channel.id} />
-        </ChannelContainer>
-        <DescriptionContainer>
-          {video ? (
-            <>
-              {video.description.split('\n').map((line, idx) => (
-                <p key={idx}>{line}</p>
-              ))}
-            </>
-          ) : (
-            <>
-              <DescriptionPlaceholder width={700} />
-              <DescriptionPlaceholder width={400} />
-              <DescriptionPlaceholder width={800} />
-              <DescriptionPlaceholder width={300} />
-            </>
-          )}
-        </DescriptionContainer>
-        <LicenseContainer>
-          {video ? (
-            <>
-              <p>
-                License:{' '}
-                {video.license.type.__typename === 'KnownLicense' ? (
-                  <a href={video.license.type.url || ''} target="_blank" rel="noopener noreferrer">
-                    {video.license.type.code}
-                  </a>
-                ) : (
-                  video.license.type.content
-                )}
-              </p>
-              {video.license?.attribution ? <p>Attribution: {video.license.attribution}</p> : null}
-            </>
-          ) : (
-            <Placeholder height={12} width={200} />
-          )}
-        </LicenseContainer>
-        <MoreVideosContainer>
-          <MoreVideosHeader>
-            {video ? `More from ${video.channel.handle}` : <Placeholder height={23} width={300} />}
-          </MoreVideosHeader>
-          <InfiniteVideoGrid ready={!loading} channelId={video?.channel.id} showChannel={false} />
-        </MoreVideosContainer>
-      </InfoContainer>
-    </Container>
+    <ViewWrapper>
+      <Container>
```

Comment:
Since we already have a wrapper for the full view here I think we can make the `Container` be actually `styled(ViewWrapper`? This way we can reduce nesting

Reply:
Or we can even call it `StyledViewWrapper` to be consistent